dnsapi/dns_freedns.sh

```diff
@@ -239,7 +252,7 @@ _freedns_login() {
   url="https://freedns.afraid.org/zc.php?step=2"
   
   _debug "Login to FreeDNS as user $username"
-  # Not using acme.sh _post() function because I need to capture the cookies.
+  #TODO Not using acme.sh _post() function because I need to capture the cookies.
```

Comment:
You should use `_post()` method, then read the cookie headers from `$HTTP_HEADER` file.

It can cross wget and curl.